### PR TITLE
Final changes for GitHub App

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

Re add `origin` for tracking of results in adviser syncs.

### CODE REVIEW

File: openshift/qeb-hwt-template.yaml [hunk 1]
```diff
@@ -34,6 +34,10 @@ parameters:
     description: "GitHub App Installation ID"
     displayName: "Installation ID"
     required: true
+  - name: ORIGIN
```
Comment (goern) [inline openshift/qeb-hwt-template.yaml:37]: I thought we have agreed that the ORIGIN is the HEAD_REPO_URL?

@fridex

**Review by goern — CHANGES_REQUESTED** [review]

File: openshift/qeb-hwt-template.yaml (hunk 1 above)
Comment (fridex) [reply openshift/qeb-hwt-template.yaml:37]: Yes, it is. But I thought we will still propagate origin (set to `HEAD_REPO_URL`) as that's required in the database to have stacks in a quaryable form agnostic to github/gitlab/...

File: app.py [hunk 2]
```diff
@@ -59,7 +59,7 @@
 CHECK_RUN_NAME = "Thoth: Advise (Developer Preview)"
 
 # no trailing / !
-ADVISE_API_URL = os.getenv("ADVISE_API_URL", "https://khemenu.thoth-station.ninja/api/v1/advise/python",)
+ADVISE_API_URL = os.getenv("ADVISE_API_URL", "https://khemenu.thoth-station.ninja/api/v1/advise/python/adviser_id",)
```
Comment (goern) [inline app.py:62]: is `adviser_id` part of the URL path?!

Comment (pacospace) [reply app.py:62]: it is required, if you want to use `urljoin`: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin. It's going to be substituted by the function.

Comment (goern) [reply app.py:62]: oh cool, didnt know that

**Pull request merged**